lib/install/vchlog/vchlogger.go

```diff
@@ -50,7 +50,7 @@ func Init() {
 func Run() {
 	sig := <-signalChan
 	// suffix the log file name with caller operation ID and timestamp
-	logFileName := sig.LogFileName + "_time_" + sig.Timestamp + "_op_" + sig.Operation.ID()
+	logFileName := "vic-machine" + "_" + sig.Timestamp + "_" + sig.Name + "_" + sig.Operation.ID()
```

Comment:
While we're adjusting this, maybe add a ".log" at the end? (And then add that to the search logic in the handler, so that if we ever have non-log files in the datastore directory we don't accidentally include them in the response.)

Reply:
good idea, thanks